### PR TITLE
A4A > Migrations: Implement Commission's list view

### PR DESCRIPTION
**Pull request by yashwin**
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1445

## Proposed Changes

This PR implements the Commission's list view for Migrations using dummy data. Please note the API endpoint will be integrated into this in a separate PR.

## Why are these changes being made?

* To show the Commission's list view on the Migrations page. 

## Testing Instructions

* Open the A4A live link >  Go to Migrations: Commissions and verify that the below screen is displayed
* Also, verify that the consolidated data on the top is correct based on the dummy API response.  

Screenshot 2024-11-11 at 7 31 09 PM

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx [hunk 1]
```diff
@@ -1,34 +1,44 @@
...
 
 	return (
 		<div className="consolidated-commissions">
 			<Card compact>
-				<div className="consolidated-commissions__value"> ${ data.migrationCommissions }</div>
+				<div className="consolidated-commissions__value"> ${ migrationCommissions }</div>
 				<div className="consolidated-commissions__label">
 					{ translate( 'Migration commissions expected in %(currentQuarter)s', {
-						args: { currentQuarter: getCurrentQuarter() },
+						args: { currentQuarter: `Q${ currentQuarter }` },
```
Comment (jkguidaven) [inline client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx]: Should we include the Q in the translatable string? Other locales may lack the appropriate context for what Q represents here.

**Review by jkguidaven — APPROVED** [review]
This works great. I have 1 minor comment but is non-blocker. LGTM

File: client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx (hunk 1 above)
Comment (yashwin) [reply client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx]: Thanks for the review, @jkguidaven!

Good point. I have updated it now: https://github.com/Automattic/wp-calypso/pull/96234/commits/d37ea62237d262de1ffec657d985b5a598138de4

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/17000976

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @yashwin for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/17000976).

**Pull request merged**